pkg/common/security/usergroup_test.go

```diff
@@ -208,9 +311,11 @@ func TestConvertUGI(t *testing.T) {
 	if err != nil {
 		t.Errorf("known user, no groups, convert should not have failed: %v", err)
 	}
+	testCache.lock.Lock()
 	if ug.User != "testuser1" || len(ug.Groups) != 2 || ug.resolved == 0 || ug.failed {
 		t.Errorf("User 'testuser1' not resolved correctly: %v", ug)
 	}
+	testCache.lock.Unlock()
```

Comment:
This lock/unlock pair can be removed. `ConvertUGI()` returns a copy and internally uses a mutex to access the map.